### PR TITLE
Feature/list and read

### PR DESCRIPTION
**Pull request by jsebastiansalazar7**

### CODE REVIEW

File: notes-app/notes.js [hunk 1]
```diff
@@ -1,8 +1,27 @@
...
+    const notes = loadNotes()
+    const noteToRead = notes.filter((note) => note.title === title)
+
+    if (noteToRead.length > 0) {
+        console.log(chalk.bold.green(title))
+        console.log(chalk.inverse(noteToRead[0].body))
+    } else {
+        console.log('The requested note does not exist')
+    }
+
```
Comment (valensag) [inline notes-app/notes.js:16]: Nit: Remove additional spaces

File: notes-app/notes.js [hunk 2]
```diff
@@ -1,8 +1,27 @@
...
+}
+
+const listNotes = function() {
+    const notes = loadNotes()
+    notes.forEach(element => {
+        console.log(chalk.bold.green(element.title))
+        console.log(chalk.inverse(element.body + "\n"))
+    });
```
Comment (valensag) [inline notes-app/notes.js:24]: It would be cool if you use business language. I find element too generic

**Review by valensag — CHANGES_REQUESTED** [review]

File: notes-app/notes.js (hunk 1 above)
Comment (jsebastiansalazar7) [reply notes-app/notes.js:16]: Yep, will do

File: notes-app/notes.js (hunk 2 above)
Comment (jsebastiansalazar7) [reply notes-app/notes.js:24]: agree!

File: notes-app/notes.js (hunk 1 above)
Comment (jsebastiansalazar7) [reply notes-app/notes.js]: done!

File: notes-app/notes.js (hunk 2 above)
Comment (jsebastiansalazar7) [reply notes-app/notes.js]: done

**Pull request merged**